Experiments/Experiments/FeatureFlag.swift

```diff
@@ -58,6 +58,10 @@ public enum FeatureFlag: Int {
     ///
     case hubMenu
 
+    /// Display the Fees refund section
+    ///
+    case refundFees
```

Comment:
Since you are shipping the full feature as it is, do we need this feature flag? Is there more functionality missing?

Reply:
Oh, I think there are no missing functionalities! The Feature Flag was created to support multiple PRs but I changed my mind when verified that it wouldn't be a large changelog after doing all needed implementations. Great catch, thanks! Fixed here: https://github.com/woocommerce/woocommerce-ios/pull/5663/commits/c1d20d78cd6a15fcdc1a8af5dadf30d6ccb1499f